### ISSUE TITLE
训练后的模型比使用的Backbone model大差不多两倍是正常的吗？

### ISSUE THREAD

**Issue by mojiayuan** [JDAI-CV/fast-reid#286]
比如使用resnet50_ibn_a预训练模型大小90多M,训练后的模型大小翻了两倍。

**Comment by L1aoXingyu** [comment]
保存的 `state_dict` 不仅包含模型的参数，还包含 `optimizer` 和 `scheduler` 等等。

**Comment by mojiayuan** [comment]
> 保存的 `state_dict` 不仅包含模型的参数，还包含 `optimizer` 和 `scheduler` 等等。

哦哦，谢谢啦

**Comment by mojiayuan** [comment]
> 保存的`state_dict`至少包含模型的参数，还包含`optimizer`和`scheduler`等等。
